src/layouts/ResourceRoom/ResourceRoom.stories.tsx

```diff
@@ -50,6 +51,34 @@ Empty.parameters = {
   msw: {
     handlers: {
       resourceRoomData: buildResourceRoomData([]),
+      resourceRoomName: buildResourceRoomName({ resourceRoomName: "" }),
+    },
+  },
+}
+Empty.decorators = [
+  (Story) => {
+    return (
+      <MemoryRouter
+        initialEntries={[
+          `/sites/storybook/resourceRoom/${MOCK_RESOURCE_ROOM_NAME}`,
+        ]}
+      >
+        <Route path="/sites/:siteName/resourceRoom/">
+          <Story />
+        </Route>
+      </MemoryRouter>
```

Comment:
could we shift this into the decorator above? it'd still apply to this and it'd be neater